.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Feature request
+about: Suggest a new feature for hypothesis' browser-extension.
+title: 'Feature request'
+labels: 'feature_request'
+assignees: ''
+---
+
+<!--
+Thank you for requesting a feature for the Hypothesis browser-extension!
+
+In order to make it easy for us to help you, please note the following:
+
+1. Only report **feature_request** with the Hypothesis browser-extension here. If you want to
```

Comment:
I suggest this bullet point can be removed for the same reasons as in my comment on the bug report template above.